native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -335,7 +339,7 @@ void EventDispatcher::dispatchDestroyWindowEvent() {
 void EventDispatcher::dispatchRecreateWindowEvent() {
 #if CC_PLATFORM == CC_PLATFORM_WINDOWS
     EventDispatcher::dispatchCustomEvent(EVENT_RECREATE_WINDOW, 1,
-                                         reinterpret_cast<void *>(CC_GET_PLATFORM_INTERFACE(ISystemWindow)->getWindowHandle()));
+                                         reinterpret_cast<void *>(CC_GET_MAIN_SYSTEM_WINDOW()->getWindowHandle()));
```

Comment:
Only main window will trigger close event?

Reply:
For the time being